### PR TITLE
docs(getting-started): adding clarifications for easily missed steps

### PR DESCRIPTION
**Pull request by sevilayha**
Just some clarifications and making sure the instructions on adding the detail component are clearer.

### CODE REVIEW

File: docs/content/getting-started.md [hunk 1]
```diff
@@ -51,37 +50,33 @@ Let's start with the contents of `index.html`:
...
 This is a pretty typical angular app, except the `router-view-port` directive.
-`router-view-port` is like `ng-view`; it's a placeholder for part of your app loaded
-dynamically based on the route configuration.
+`router-view-port` is like `ng-view`; it's a placeholder for part of your app loaded dynamically based on the route configuration.
 
 So how do we configure the app? Let's open `app.js` and find out. Add this to the file:
 
 ```js
-angular.module('app', ['ngNewRouter'])
+angular.module('myApp', ['ngNewRouter'])
```
Comment (PascalPrecht) [inline docs/content/getting-started.md]: We should decide here if we want to use `app` or `myApp`. This PR introduces `myApp` but still uses `app` later on

File: docs/content/getting-started.md [hunk 2]
```diff
@@ -189,11 +181,22 @@ function DetailController ($routeParams) {
...
+Now that our detail component has been created, it just needs to be loaded in `index.html` and added to the `app` module like the `home` component:
+
+```html
+...
+<script src="./components/detail/detail.js"></script>
 ```
 
-Then we'd have to wire up the controller by adding a script tag and making out `app` module depend on `app.detail`.
+```js
+angular.module('app', ['ngNewRouter', 'app.home', 'app.detail'])
```
Comment (PascalPrecht) [inline docs/content/getting-started.md]: Here, it still uses `app`

**Comment by PascalPrecht** [comment]
Okay, I gonna squash + rebase your commits + some changes, on latest master once I've merge https://github.com/angular/router/pull/122

**Comment by sevilayha** [comment]
Sounds good. Thanks.

**Pull request closed without merging**